### ISSUE TITLE
Anyone using volume controls in tvOS?

### ISSUE THREAD

**Issue by postlund** [postlund/pyatv#489]
**What's on your heart?**
I want to add "feature" support for volume when implementing #268, i.e. make it possible to ask `pyatv` if current device supports `volume_up` and `volume_down`. Unfortunately, my setup doesn't support controlling sound via CEC so I cannot test it myself. Is there anyone out willing to help out?

What I need is debug logs, i.e. running:
```shell
$ atvremote --debug ... playing
```

*But* a small change must be made to `pyatv` first. In this line:

https://github.com/postlund/pyatv/blob/778257f9b3e72a2a6fe2534b956893693d255e68/pyatv/mrp/messages.py#L81

`volume=False` must be changed to `True` instead.

**Comment by jdebardi** [comment]
I have CEC volume on 2 4K ATVs and happy to test

**Comment by aptonline** [comment]
My Apple remote changes the external speakers via IR I’m afraid.

**Comment by jdebardi** [comment]
What version do you want/need me to run the tests on (and how do I install required version if not a released one?)?

**Comment by postlund** [comment]
@jdebardi I think it's easiest if you run from `master` but version is not that important. Making the change I mentioned above is the important part, I hope that is enough.

**Comment by postlund** [comment]
I've made it simple and merged the necessary change (#490). So it should be possible to install `pyatv` from git with:

    pip install git+https://github.com/postlund/pyatv.git

**Comment by postlund** [comment]
I noticed that I got this message in #204:

```
DEBUG:pyatv.mrp.connection:<< Receive: Protobuf=type: VOLUME_CONTROL_AVAILABILITY_MESSAGE
priority: 0
[volumeControlAvailabilityMessage] {
  volumeControlAvailable: true
  volumeCapabilities: 1
}
```

This is was I was expecting, so I guess I can use that when implementing this feature.
